lisa/tools/aria.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import PurePosixPath
+from typing import TYPE_CHECKING
+
+from lisa.base_tools import Wget
+from lisa.executable import Tool
+from lisa.tools.make import Make
+from lisa.tools.tar import Tar
+
+if TYPE_CHECKING:
+    from lisa.operating_system import Posix
+
+
+class Aria(Tool):
+    _DOWNLOAD_LOCATION = "https://github.com/q3aql/aria2-static-builds/releases/download/v1.35.0/aria2-1.35.0-linux-gnu-64bit-build1.tar.bz2"  # noqa: E501
+    _DOWNLOAD_TAR_NAME = "aria2-1.35.0-linux-gnu-64bit-build1.tar.bz2"
+    _DOWNLOAD_NAME = "aria2-1.35.0-linux-gnu-64bit-build1"
+
+    @property
+    def command(self) -> str:
+        return "aria2c"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = self.node.os  # type: ignore
+        try:
+            posix_os.install_packages(["aria2"])
+        except Exception as identifier:
+            self._log.debug(f"Failed to install aria2 from apt: {identifier}")
+
+        if not self._check_exists():
+            self._log.debug("Installing aria2 from source")
+            downloaded_file_path = self.node.tools[Wget].get(
+                self._DOWNLOAD_LOCATION,
+                file_path=str(self.node.working_path),
+                filename=self._DOWNLOAD_TAR_NAME,
+                force_run=True,
+            )
+            self.node.tools[Tar].extract(
+                file=downloaded_file_path, dest_dir=str(self.node.working_path)
+            )
+
+            # make and install aria2
+            self.node.tools[Make].make_install(
+                cwd=self.node.working_path / self._DOWNLOAD_NAME
+            )
+
+        return self._check_exists()
+
+    def get(
+        self,
+        url: str,
+        file_path: str = "",
+        filename: str = "",
+        overwrite: bool = True,
+        sudo: bool = False,
+        force_run: bool = False,
+        num_connections: int = 2,
+        timeout: int = 600,
+    ) -> str:
+
+        if file_path:
+            # create folder when it doesn't exist
+            self.node.shell.mkdir(PurePosixPath(file_path), exist_ok=True)
+        else:
+            file_path = str(self.node.working_path)
+
+        download_path = f"{file_path}/{filename}"
+        download_pure_path = self.node.get_pure_path(download_path)
+        if overwrite and self.node.shell.exists(download_pure_path):
+            self.node.shell.remove(download_pure_path, recursive=True)
+            force_run = True
```

Comment:
seems this tool has `--allow-overwrite` this parameter, if so, please use this parameter.